core/src/test/scala/shapeless/serialization.scala

```diff
@@ -228,7 +234,7 @@ object SerializationTestDefns {
 
 }
 
-class SerializationTests {
+class SerializationTests extends Serializable {
   import SerializationTestDefns._
```

Comment:
I fixed similar things in 2.2.5 here: https://github.com/milessabin/shapeless/commit/78def231b91b45786aa9676fc1398b868056ac04. Is it possible that you've lost this in the update?